src/OrchardCore.Modules/OrchardCore.ContentPreview/Controllers/PreviewController.cs

```diff
@@ -63,10 +66,6 @@ public async Task<IActionResult> Render()
             var contentItemId = Request.Form["PreviewContentItemId"];
             var contentItemVersionId = Request.Form["PreviewContentItemVersionId"];
 
-            // Unique contentItem.Id that only Preview is using such that another
-            // stored document can't have the same one in the IContentManagerSession index
-
-            contentItem.Id = -1;
```

Comment:
We still need this because of the above comments, it is necessary when we use a preview part / aspect to redirect (through the `PreviewStartupFilter`) to the preview path e.g. a razor page that will query and need to retrieve the previewed content item through the `IContentManagerSession`.

Reply:
Returned ;)